index.html

```diff
@@ -47,13 +47,39 @@ <h1>Rocket launcher</h1>
                     <span class="label">Lives:</span>
                     <span id="lives-remaining-value" class="value">33</span>
                 </div>
+                <div class="btnContainer">
+                    <div class="pause btn ">
+                        <div class="btnBackgroundDiv">
+                            <img src="./assets/circle.png" alt="restart button">
+                        </div>
+                        <div class="insideBtnDiv">
+                            <img class="insideBtn" src="./assets/pause.png">
+                        </div>
+                    </div>
```

Comment:
1. For any interaction, action on the page, use the `button` tag
2. Try using as few elements as possible
3. Use `:before` and `:after` when possible
4. Good looking buttons btw :)))

Reply:
updated